controllers/office.js

```diff
@@ -1,46 +1,27 @@
 const OfficeModel = require('../models/office')
 
-module.exports.get = function get (req, res) {
-  OfficeModel.find({}).exec(function (err, value) {
-    if (err) return res.status(500).send()
-    res.json({value})
-  })
+module.exports.get = async function get (req, res) {
+  const data = await OfficeModel.find({})
+  res.json({data})
 }
 
-module.exports.getOne = function getOne (req, res) {
-  OfficeModel.findOne({ _id: req.params.id }, req.body)
-  .exec(function (err, value) {
-    if (err) return res.status(500).send()
-    res.json({value})
-  })
+module.exports.getOne = async function getOne (req, res) {
+  const data = await OfficeModel.findOne({ _id: req.params.id }, req.body)
+  res.json({data})
 }
 
-module.exports.post = function post (req, res) {
+module.exports.post = async function post (req, res) {
   const newOffice = new OfficeModel(req.body)
-  newOffice.save()
-    .then(function (value) {
-      res.json({
-        _id: value._id
-      })
-    }).catch(function (err) {
-      return res.status(500).send(err)
-    })
+  const data = await newOffice.save()
+  res.json({ _id: data._id })
 }
 
-module.exports.put = function put (req, res) {
-  OfficeModel.findOneAndUpdate({ _id: req.params.id }, req.body)
-    .then(function (value) {
-      res.send()
-    }).catch(function (err) {
-      return res.status(500).send(err)
-    })
+module.exports.put = async function put (req, res) {
+  await OfficeModel.findOneAndUpdate({ _id: req.params.id }, req.body)
+  await res.send()
```

Comment:
res.send() is a syncronus function, and is the last one, you dont need to make it awaitable